R/estimate_R.R

```diff
@@ -190,30 +198,30 @@
 #'
 #' ## estimate the reproduction number (method "non_parametric_si")
 #' ## when not specifying t_start and t_end in config, they are set to estimate
-#' ## the reproduction number on sliding weekly windows                          
-#' res <- estimate_R(incid = Flu2009$incidence, 
+#' ## the reproduction number on sliding weekly windows                      
```

Comment:
at the end of the examples could you please add one example of how to use back imputation - if possible on one of the existing datasets in the package? @abriz97 . Not totally necessary but will definitely improve visibility of the method I think

Reply:
answered in commit f00b67e
 [add] example of back_imputation in roxy doc